.github/workflows/tests.yml

```diff
@@ -25,10 +25,27 @@ jobs:
         sudo apt-get update
         sudo apt-get -y install libxml2-dev libxmlsec1-dev libxmlsec1-openssl libexempi-dev
 
+    - run: docker-compose -f ../.github/docker-compose.yml up -d
+
+    - name: Cache pip
```

Comment:
you can enable cache via [setup-python](https://github.com/actions/setup-python#caching-packages-dependencies)